vscode/src/extension.ts

```diff
@@ -172,6 +189,14 @@ async function findExecutables(): Promise<Executables> {
     lspCommand = configLspCommand;
   }
 
+  if (jagExec === null) {
+    const jagResult = run("jag", ["version"]);
```

Comment:
You might be able to use 'jag setup --check' by now.

Reply:
Will look at this in a different PR.